omod/src/main/java/org/openmrs/module/owa/web/controller/InstallAppController.java

```diff
@@ -0,0 +1,134 @@
+package org.openmrs.module.owa.web.controller;
+
+/*
+ * Copyright (c) 2004-2014, University of Oslo
```

Comment:
Did you copy this code from University of Oslo?

Reply:
@dkayiwa No I didn't, I saw it in the code for AddAppController and imagined it was an open source license like MIT